src/app/child-dev-project/attendance/attendance-calendar/attendance-calendar.component.ts

```diff
@@ -137,9 +137,11 @@ export class AttendanceCalendarComponent implements OnChanges {
       this.selectedEvent = this.records.find((e) =>
         this.selectedDate.isSame(e.date, "day")
       );
-      this.selectedEventAttendance = this.selectedEvent?.getAttendance(
-        this.highlightForChild
-      );
+      if (this.highlightForChild) {
```

Comment:
A suggestion but maybe out of scope for this.
Move the selected event box next to the calendar for big screens as there is a lot of empty space on the screen.
<img width="541" alt="attendance-calendar" src="https://user-images.githubusercontent.com/33730997/150766521-ebe319ba-f04c-42a5-a3df-d74043ec8ee3.png">